src/widgets/window.js

```diff
@@ -258,7 +259,7 @@ var FlatsealWindow = GObject.registerClass({
             showPermissionsButton = false;
         }
 
-        const isFolded = this._headerLeaflet.folded;
+        const isFolded = this._contentLeaflet.folded;
         if (isFolded) {
             showApplicationsButton = true;
             showPermissionsButton = true;
```

Comment:
Minor issue:

![Peek 2021-01-23 18-18](https://user-images.githubusercontent.com/481418/105614337-7fcd1380-5da7-11eb-93c5-322b1102c213.gif)


Would it be possible to disable the hovering effect when is not needed? Is a bit distracting when hovering over options like the above.

Codewise, it seems to me this should only be enabled for `FlatsealPermissionSwitchRow`.